game/scripts/npc/abilities/earth_spirit_magnetize.txt

```diff
@@ -19,7 +19,7 @@
 
     // Casting
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCastRange"                                    "350"
+    "AbilityCastRange"                                    "375" //OAA
```

Comment:
You should leave the AOE unchanged. People are grouped up enough to the point where you should quickly be able to spread magnetise.

Reply:
ok so does it matter than? 25 is just a nice QOL fix

Reply:
like you're literally asking for earth spirit buffs and now you don't want them?

Reply:
Why make this change though? It serves no purpose. The issue with magnetise is that it deals no damage.

Reply:
The issue with Magnetise is that its easily dispellable. So any change to Magnetise is useless so whatever.